lib/std/hash/rapidhash.zig

```diff
@@ -0,0 +1,128 @@
+const std = @import("std");
+
+const readInt = std.mem.readInt;
+const assert = std.debug.assert;
+const expect = std.testing.expect;
+const expectEqual = std.testing.expectEqual;
+
+const RAPID_SEED: u64 = 0xbdd89aa982704029;
+const RAPID_SECRET: [3]u64 = .{ 0x2d358dccaa6c78a5, 0x8bb84b93962eacc9, 0x4b33a62ed433d4a3 };
+
+pub const RapidHash = struct {
+    pub fn hash(seed: u64, input: []const u8) u64 {
+        return rapidhash_internal(input, input.len, seed, &RAPID_SECRET);
```

Comment:
Why is the implementation not in here but calling another function?

Reply:
I wrote my first version as a more direct port where they have a seed vs no_seed public function.

I can inline that if we don't want a no-seed version.

Reply:
Please inline it, I don't see the need for a no-seed version. 
Also, I don't see why we couldn't benefit from Zig's file-level structs here. Move the `hash` function into the file scope and rename the file to `RapidHash.zig`.

Reply:
Roger